### PR TITLE
Prioritizing the grind

### PR DESCRIPTION
**Pull request by drainpip**

### CODE REVIEW

File: content/blog/prioritizing-the-grind/index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+);
+```
+<!-- CODE BLOCK - END -->
+<!-- prettier-ignore-end -->
+
+Know why we're building the thing, share the why of how we're building the thing
+
+Flow vs red tape
+
+Counting points vs demoing features
```
Comment (treighton) [inline content/blog/prioritizing-the-grind/index.md]: Are these points you'll elaborate on?

I think points vs demoing features could be a whole section

**Review by treighton — COMMENTED** [review]
This is great. I like the pseudo code bits; they provide a bit of levity and brevity.

File: content/blog/prioritizing-the-grind/index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Fostering this environment will empower your team to continue this collaboration without being told to do so. People will automatically jump in to help those that need it, or offer their support to teach those that want to learn. This starts to become a fun place to go to work every day. It creates a cycle of more capable and happy individuals which leads to the feeling of rampant creation anyone that's been on one of these teams will recognize immediately.
+
+There is an upper limit to how expensive collaboration is on a given team. Communication scales exponentially for every new person on a team which can cause everything to grind to a halt. There are also best judgement calls you will make regularly about critical fires that come up that take precedence, as long as this is not the norm this is totally normal. Efficiency is important, but not if it kneecaps your team every day away from what will help them grow and work on what they feel is important.
```
Comment (treighton) [inline content/blog/prioritizing-the-grind/index.md]: Learning how to find the upper limit, or how to notice when you're approaching it is a tough skill to learn. Do you have "flags" that you watch for? Could those be made into a pseudo code?

```
while(
engagement && perceivedProductivity &&
learning &&
deliveringValue 
) 
```

Or 🤷‍♂️

File: content/blog/prioritizing-the-grind/index.md (hunk 1 above)
Comment (drainpip) [reply content/blog/prioritizing-the-grind/index.md]: Yeah the short-hand stuff is just for me to remember what I was thinking when I first thought of it. Most of the things I put up can be expanded, in fact most of my blog posts are just expansions of previous paragraphs or sentences from older ones...

File: content/blog/prioritizing-the-grind/index.md (hunk 2 above)
Comment (drainpip) [reply content/blog/prioritizing-the-grind/index.md]: Good question, I'll think of ways to talk more practically about it.

File: content/blog/play-the-long-game/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (drainpip) [inline content/blog/play-the-long-game/index.md:1]: Changes in this file were to separate out the focus of this post (long-term thinking vs. daily grind)

**Pull request merged**